docs/UserGuide.md

```diff
@@ -54,20 +54,37 @@ ContactsForGood (CFG) is a **desktop app for managing contacts, optimized for us
 * Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
   e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
 
+* Each parameter is preceded by a **prefix**, which indicates the type of data the parameter represents.  
+  For example:
+    - `n/` is the prefix for `NAME`, as in `add n/John Doe`.
+    - `p/` is the prefix for `PHONE_NUMBER`, as in `add p/12345678`.
+
+* Any unrecognized parameter's prefix will be treated as part of the value for the preceding valid prefix. This means that care should be taken to use only the correct prefixes when entering commands to avoid unexpected behavior.
+    For example:
+    - Correct: `add n/John Doe p/12345678`
+    - Incorrect: `add n/John Doe xyz/12345678` (Here, `xyz/12345678` will be treated as part of the name value, which
+      will result in an error)
+
 * Items in square brackets are optional.<br>
   e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
 
 * Items with `…`​ after them can be used multiple times including zero times.<br>
   e.g. `[t/TAG]…​` can be used as ` ` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
 
 * An `INDEX` refers to the person at the index number shown in the **last displayed person list**.<br>
```

Comment:
Should we dumb it down for the user even more and include a screenshot of where they can find the index? As it may be unclear to someone who is using the product for the first time.

@seah-minlong Thoughts?

Reply:
Cannot be bah... numbered list. Considering our ta are ngo administrators too, they should understand when they see the numbers